db/migrate/20170217185047_create_api_keys.rb

```diff
@@ -0,0 +1,9 @@
+class CreateApiKeys < ActiveRecord::Migration[5.0]
+  def change
+    create_table :api_keys do |t|
+      t.string :api_key_public
+      t.string :api_key_hash
```

Comment:
is `:api_key_hash` the same as the private fragment of `params[:api_key]` or is there some transformation between the database attribute and the value used in the controller?